docs/installation.md: update centos and rhel steps

lsm5 opened the pull request:
CentOS 7 and CentOS Stream 8 are no longer active. Also, for RHEL, it's best to point to the official RHEL documentation instead of duplicating instructions here.

TomSweeneyRedHat commented:
@rhatdan PTAL and merge if it LGTY

TomSweeneyRedHat commented:
And after pushing the first commit, I pulled up Ed's post and realized it was missing a readmore and then spotted  word that needed uppercasing.  I've added those too.

rhatdan commented:
LGTM

lsm5 commented:
waiting on confirmation for the rhel installation doc link.

rhatdan commented:
LGTM

TomSweeneyRedHat commented:
The links and the rest LGTM.  @lsm5 once you're content with the RHEL links, feel free to merge.

lsm5 commented:
The redhat kbase link is now ready to go. So, this PR is ready to merge as well.

Thanks @GabiNecasova for the kbase update.

lsm5 commented:
FYI, the redhat kbase article will need a redhat login and a subscription (free developer subscriptions work). So, I don't think that should be a problem for anyone on a rhel env.

lsm5 commented:
> FYI, the redhat kbase article will need a redhat login and a subscription (free developer subscriptions work). So, I don't think that should be a problem for anyone on a rhel env.

Update: the kbase article is now visible to everyone, no need for login.

lsm5 committed:
docs/installation.md: update centos and rhel steps

CentOS 7 and CentOS Stream 8 are no longer active. Also, for RHEL, it's
best to point to the official RHEL documentation instead of duplicating
instructions here.

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

lsm5 commented:
made a small additional change: `s/yum/dnf/` for centos.

rhatdan commented:
LGTM
